skimage/restoration/_rolling_ball.py

```diff
@@ -5,29 +5,30 @@
 
 
 def rolling_ball(image, *, radius=100, kernel=None, nansafe=False, num_threads=None):
-    """Estimate background intensity by rolling/translating a kernel.
+    """Estimate background intensity by translating a kernel over an image.
 
-    This rolling ball algorithm estimates background intensity for a
-    ndimage in case of uneven exposure. It is a generalization of the
-    frequently used rolling ball algorithm [1]_.
+    This function estimates the background intensity of an n-dimensional
+    image. Typically, it is useful for background subtraction in case of
+    uneven exposure. It is a generalization of the well-known rolling-ball
+    algorithm [1]_.
 
     Parameters
     ----------
     image : ndarray
         The image to be filtered.
     radius : int, optional
-        Radius of a ball-shaped kernel to be rolled/translated in the image.
-        Used if ``kernel = None``.
+        Radius of the ball-shaped kernel to be translated over the
+        image. Used only if `kernel` is ``None``.
     kernel : ndarray, optional
-        The kernel to be rolled/translated in the image. It must have the
-        same number of dimensions as ``image``. Kernel is filled with the
-        intensity of the kernel at that position.
```

Comment:
@FirefoxMetzger I originally tried fixing the sentence "Kernel is filled with the intensity of the kernel" but then I thought these details would fit better in the Notes.